### PR TITLE
docs: correct typos and ensure consistency

### PR DESCRIPTION
**Pull request by wtlin1228**
# What is it?

- [ ] Feature / enhancement
- [ ] Bug
- [x] Docs / tests

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

# Use cases and why

- 1. One use case
- 2. Another use case

# Checklist:

- [x] My code follows the [developer guidelines of this project](https://github.com/BuilderIO/qwik/blob/main/CONTRIBUTING.md)
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

File: packages/docs/src/routes/docs/(qwikcity)/action/index.mdx [hunk 1]
```diff
@@ -183,13 +184,16 @@ This allows actions to access the request headers, cookies, url and environment
 import { routeAction$ } from '@builder.io/qwik-city';
 
 // The second argument of the action is the `RequestEvent` object
-export const useProductRecommendations = routeAction$(async (_data, requestEvent) => {
-  console.log('Request headers:', requestEvent.headers);
-  console.log('Request cookies:', requestEvent.cookie);
-  console.log('Request url:', requestEvent.url);
-  console.log('Request params:', requestEvent.params);
-  console.log('MY_ENV_VAR:', requestEvent.env.get('MY_ENV_VAR')));
```
Comment (wtlin1228) [inline packages/docs/src/routes/docs/(qwikcity)/action/index.mdx:191]: There is an extra `)` at the end of this line.

File: packages/docs/src/routes/docs/(qwikcity)/action/index.mdx [hunk 2]
```diff
@@ -107,7 +107,7 @@ export default component$(() => {
   return (
     <section>
       <button
-        onClick={async () => {
+        onClick$={() => {
```
Comment (manucorporat) [inline packages/docs/src/routes/docs/(qwikcity)/action/index.mdx:110]: the `async` is still needed

File: packages/docs/src/routes/docs/(qwikcity)/action/index.mdx [hunk 3]
```diff
@@ -20,7 +20,7 @@ Actions can be declared using the `routeAction$()` or `globalAction$()` exported
 import { component$ } from '@builder.io/qwik';
 import { routeAction$, Form } from '@builder.io/qwik-city';
 
-export const useAddUser = routeAction$(async (data, requestEvent) => {
+export const useAddUser = routeAction$(async (data) => {
```
Comment (manucorporat) [inline packages/docs/src/routes/docs/(qwikcity)/action/index.mdx:23]: even if `requestEvent` not used, i would rather leave it there. it's good for Qwik AI to know what to expect in every argument

**Review by manucorporat — CHANGES_REQUESTED** [review]

File: packages/docs/src/routes/docs/(qwikcity)/action/index.mdx (hunk 3 above)
Comment (wtlin1228) [reply packages/docs/src/routes/docs/(qwikcity)/action/index.mdx]: Got it!

**Review by manucorporat — APPROVED** [review]

**Pull request merged**